pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -876,3 +879,104 @@ func (c updateRulesOrderInGroup) Exec(sess *xorm.Session, migrator *migrator.Mig
 	}
 	return nil
 }
+
+func AddExternalAlertmanagerToDatasourceMigration(mg *migrator.Migrator) {
```

Comment:
Could we implement this migration in a separate file or package, rather than appending it to the `ualert.go` migration file?

The vast majority of the code in `ualert.go` is around the Legacy Alerting -> Unified Alerting migration. It's not meant to be an umbrella package for all alerting-related migrations, rather most of this code is one very big migration that already happened and should be "frozen" unless we are fixing a bug in that area.